### PR TITLE
Add hover effect to user-card

### PR DESCRIPTION
**Pull request by devXprite**
## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### Preview :

https://user-images.githubusercontent.com/80192140/193983868-4cab5eaf-8caf-4c3c-8466-6f77e8ae3e88.mp4

### CODE REVIEW

**Comment by Adarsh-jaiss** [comment]
- @vikasganiga05 please review this

File: style.css [hunk 1]
```diff
@@ -13,6 +13,13 @@
   background-color: var(--borderColor);
   border-radius: 10px;
   border: 1px solid var(--accent);
+  transition: all 0.3s;
+}
+
+.user-card:hover {
+  border-width: 2px;
```
Comment (vikasganiga05) [inline style.css:20]: My suggestion is to remove this because other cards shift their position.
```suggestion

```

File: style.css [hunk 2]
```diff
@@ -13,6 +13,13 @@
...
+  transform: scale(1.05);
+  filter: brightness(1.3);
```
Comment (vikasganiga05) [inline style.css:22]: This does not look good on the light theme.

**Review by vikasganiga05 — CHANGES_REQUESTED** [review]
When the user hovers on one card the other cards shift their position. I suggested changes below. Please check them out.

**Comment by devXprite** [comment]
@vikasganiga05  I Did the suggested changes!!

**Review by vikasganiga05 — APPROVED** [review]
It looks good to me. Thanks for your contribution :tada:

**Pull request merged**